packages/action-tracker/src/index.ts

```diff
@@ -15,14 +15,29 @@ import {
 export const buildEmbraceMiddleware = () => {
   return (store: IMiddleware<IDispatch, any>) =>
     (next: IDispatch<IAnyAction>) =>
-    (action: IAnyAction) => {
+    async (action: IAnyAction) => {
       if (!next || typeof next !== "function") {
         console.warn(
           "[Embrace] The state managment middleware was not applied succesfully",
         );
         return 0;
       }
       const startTime = new Date().getTime();
+      const attributes = {
+        name: action.type.toString().toUpperCase(),
+        "emb.send_immediately": true,
```

Comment:
discussed offline (https://embrace-io.slack.com/archives/C05NTFHCFBJ/p1721164675230939), we can remove the  "emb.send_immediately" attribute

Reply:
I removed that attribute. What do you want to discuss offline ?  ( I cannot enter to that link)

Reply:
just that `emb.send_immediately` was an internal attribute set by the Android SDK and not something we need to worry about when using the spans API here